gen/generate.go

```diff
@@ -30,10 +30,16 @@ import (
 
 	"go.uber.org/thriftrw/compile"
 	"go.uber.org/thriftrw/internal/plugin"
+	"go.uber.org/thriftrw/plugin/api"
 
 	"go.uber.org/multierr"
 )
 
+// CodeGenerator lists possible code generators for a plugin.
+type CodeGenerator struct {
```

Comment:
I think Name() would be a carryover from internal/plugin.Handle

Reply:
Looks like name is unused at this level.

internal/plugin.Handle is a reference to a specific external ThriftRW plugin. The name is probably meant for debugging.

At this layer, we really just care about the different code generators provided by the plugins, if at all. Their names are less important. (Note that the plugin.Handle is still in scope in main.go where the  name may be relevant.)